bin/www

```diff
@@ -45,18 +43,16 @@ function onError(error) {
     throw error
   }
 
-  const bind = typeof port === "string" ? "Pipe " + port : "Port " + port
+  const bind = typeof port === "string" ? `Pipe ${port}` : `Port ${port}`
```

Comment:
we should file an issue to refactor and get rid of this ._.

Reply:
Filed an issue here: https://github.com/isomerpages/isomercms-backend/issues/387